### ISSUE TITLE
NTR: CT or MRI: spinal extradural cysts (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8926]
New term request CT or MRI: spinal extradural cysts (elthat_100823185853): Epidural  UMLS: 0344486

**Comment by pnrobinson** [comment]
@MickeySegal
is this Spinal Extradural Arachnoid Cyst (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5110915/)

**Comment by MickeySegal** [comment]
I'm having trouble making sense of that because it says "Type I cysts are composed of extradural arachnoid cysts (EAC) without nerve root fibers, Type II cysts are composed of EAC with nerve root fibers, and Type III cysts are composed of intradural meningeal cyst"

We used our finding for https://www.ncbi.nlm.nih.gov/books/NBK582154/ "Tarlov cysts are perineural cysts between the perineurium and endoneurium, arising near the dorsal root ganglion.[1] They can occur anywhere in the spine but most frequently around the sacral nerve roots, with S2 being the most common level.[2] The cysts are usually filled with cerebrospinal fluid but may contain blood if there is a hemorrhagic complication"

https://www.ncbi.nlm.nih.gov/books/NBK1457/ "Spinal extradural arachnoid cysts"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000849
- label: Spinal extradural arachnoid cyst
- parents: [Abnormal spinal cord morphology (HP:0002143)](https://hpo.jax.org/app/browse/term/HP:0002143)
- pmid: PMID:24278289
- def: Spinal extradural arachnoid cyst is a cyst in the spinal canal that protrudes into the epidural space from a defect in the dura mater.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8926
- synonyms: SEDAC(NONE)
